Jenkinsfile-smoke.groovy

```diff
@@ -9,7 +9,7 @@ if (env.CHANGE_ID) {
     execSmokeTest (
         ocDeployerBuilderPath: "hccm/koku",
         ocDeployerComponentPath: "hccm",
-        ocDeployerServiceSets: "hccm,platform,platform-mq",
+        ocDeployerServiceSets: "ingress,inventory,platform-mq,hccm",
```

Comment:
Also my guess is we don't actually need inventory either right? We can probably remove that too if I am correct @adberglund

Reply:
@lcouzens  I don't think we need inventory - but @adberglund would know better than me!

Reply:
Yeah, inventory can go.